index.html

```diff
@@ -207,7 +207,28 @@ <h3>
               "type": "image/jpeg"
             }]
           }
-      </pre>
+        </pre>
+        <p>
+          The following shows a <a>manifest</a> that prefers the <code>
+          minimal-ui</code> [=display modes|display mode=]
+          to <code>standalone</code>, but if minimal-ui isn't supported, fall
+          back to standalone as opposed to browser.
+        </p>
+        <pre class="example json" title="Advanced display usage manifest">
+          {
+            "name": "Recipe Madness",
```

Comment:
I'm not sure how big of a deal this particular one is, but given the recent efforts around inclusive language, it might be best to avoid the term "madness" here and perhaps just go "Recipe Book", or if you want to spice it up, "Recipe Time", "Recipe Bonanza" or "Recipe Zone", etc.